cmake/HostVerifyNuGetDescription.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
```

Comment:
If this file goes directly to Nuget description, the copyright statements can be omitted.